database/migrations/2019_04_05_072801_insert_config_adpay_event_last_exported_id.php

```diff
@@ -46,6 +46,8 @@ public function up(): void
         DB::table('configs')->updateOrInsert(
             [
                 'key' => Config::ADPAY_LAST_EXPORTED_EVENT_ID,
+            ],
```

Comment:
Was that tested and is it really needed?

Reply:
yes